sdk/resources/Azure.ResourceManager.Resources/src/Customized/DeploymentPropertiesExtended.Serialization.cs

```diff
@@ -23,7 +23,7 @@ internal static DeploymentPropertiesExtended DeserializeDeploymentPropertiesExte
             Optional<DateTimeOffset> timestamp = default;
             Optional<TimeSpan> duration = default;
             Optional<BinaryData> outputs = default;
-            Optional<IReadOnlyList<ProviderData>> providers = default;
+            Optional<IReadOnlyList<ResourceProviderData>> providers = default;
```

Comment:
I cannot generate this file and the file below. In swagger, the definition has a `Providers` property of which the type is `Provider`. I rename `Provide` to `ResourceProvider`, but still cannot generate `<ResourceProviderData>`, so I use customization here.